apps/financial-aid/web-osk/src/routes/application/Info/info.tsx

```diff
@@ -12,18 +12,21 @@ import { useRouter } from 'next/router'
 
 import useFormNavigation from '@island.is/financial-aid-web/osk/src/utils/useFormNavigation'
 
-import { NavigationProps } from '@island.is/financial-aid/shared/lib'
-import { UserContext } from '@island.is/financial-aid-web/osk/src/components/UserProvider/UserProvider'
+import {
+  getNextMonth,
+  NavigationProps,
+} from '@island.is/financial-aid/shared/lib'
 
 import { useLogOut } from '@island.is/financial-aid-web/osk/src/utils/useLogOut'
 
 const ApplicationInfo = () => {
   const router = useRouter()
-  const { setUser, user } = useContext(UserContext)
 
   const [accept, setAccept] = useState(false)
   const [hasError, setHasError] = useState(false)
 
+  const nextMonth = getNextMonth(new Date()).toLowerCase()
```

Comment:
Since the function is called getNextMonth then it should probably be hardecoded to return current month + 1 and it should probably return the month name lower cased.

Another approach we can take is make a utils function which is `getMonth(month: number)`, I would personally go that way and use that function for `currentMonth` and `nextMonth`